x-pack/plugins/cases/public/components/edit_connector/index.tsx

```diff
@@ -235,7 +236,7 @@ export const EditConnector = React.memo(
                       connectors,
                       dataTestSubj: 'caseConnectors',
                       defaultValue: selectedConnector,
-                      disabled,
+                      disabled: !userCanCrud,
```

Comment:
I think we need the form to render so that the `currentConnector` can be initialized so a readonly user can still view the previously configured connector 👍